projects/travel-tracker.md

```diff
@@ -26,14 +27,28 @@ Tuesday, Week 6 - Project due at 12PM Mountain Time.
 
 # Requirements
 
+## Workflow
+You will want to submit PRs to your accountabilibuddy to:
+
+* You must give your accountabilibuddy collaboration access to your repo.
+* You must submit *at least* 2 PRs to your accountabilibuddy for review.
+* You must wait for your accountabilibuddy to review your PRs, and allow THEM to merge any PRs you submit.
+
+It is up to you to decide what changes warrant a PR – remember we want to submit PRs that have significant changes and potential for feedback. Think about what functionality you’re struggling with or have questions about or need help with. As an accountabilibuddy, you are responsible for reviewing at least 2 PRs from your partner.
+
+**Please also tag your project manager in any PR you make to your buddy.**
```

Comment:
Speaking to the comment @robbiejaeger made about removing the requirement for a review from their accountabilibuddy, what do ya'll think about the having the project manager tagged to a PR still?  Should this also be removed, or should we require them to still tag their PM in one of the PRs?

Reply:
Good question! My vote is that we don't need them to tag their PM in their request. I've always felt it was a reasonable lift to just look through their PRs anyways and I can easily identify which one has had the review from the accountabilibuddy. I'm open for other ideas, though!